cmd/finch/virtual_machine_status_test.go

```diff
@@ -0,0 +1,126 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package main
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/golang/mock/gomock"
+	"github.com/runfinch/finch/pkg/dependency"
+	"github.com/runfinch/finch/pkg/mocks"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestNewStatusVMCommand(t *testing.T) {
+	t.Parallel()
+
+	cmd := newStatusVMCommand(nil, nil)
+	assert.Equal(t, cmd.Name(), "status")
+}
+
+func TestStatusVMAction_run(t *testing.T) {
```

Comment:
Also need nonExistent and undefined use case

Reply:
I am not sure if we really need the [two cases here](https://github.com/runfinch/finch/blob/13eeaf5aea689eb7d261173459602c6c4e899a9c/cmd/finch/virtual_machine_status.go#L54:L57) since the `lima.GetVMStatus` returns an `error` when the instance state is unknown already. I added the `nonExistent` test in [453e2dc](https://github.com/runfinch/finch/pull/83/commits/453e2dc0b6d091b19b6fab2e6a19b0ba4e1fc267) where I also had to add the `Nonexistent` status in `toVMStatus`

Reply:
https://github.com/runfinch/finch/pull/83/files#r1056636231